### PR TITLE
Extended Items Conversion

### PR DESCRIPTION
**Pull request by Nina-1474**
A bug was raised recently where reverse-suspension and ziptie frogtie could both be applied causing the player to be kneeling upside-down. This turned out to be because the item's script wasn't checking the Frogtie's prerequisites properly. Examination of other similar items showed similar incorrect copy-pasted versions, along with other inconsistencies such as where to display the requirement message.
Rather than correct these details, I've changed the items to use the standed ExtendedItem scripts which achieves the same thing. The extended script has a small change to support the Prerequisites settings without needing custom Validate functions that all do the same thing.
I tested each item and didn't find any issues or changes in behaviour other than the intended fixes.

### CODE REVIEW

File: BondageClub/Screens/Inventory/ItemPelvis/HempRope/HempRope.js [hunk 1]
```diff
@@ -3,105 +3,38 @@
...
-	}
+function InventoryItemPelvisHempRopePublishAction(C, Option) {
+	var msg = "PelvisRopeSet" + Option.Name;
+	var Dictionary = [];
+	Dictionary.push({ Tag: "SourceCharacter", Text: Player.Name, MemberNumber: Player.MemberNumber });
+	Dictionary.push({ Tag: "TargetCharacter", Text: C.Name, MemberNumber: C.MemberNumber });
+	ChatRoomPublishCustomAction(msg, true, Dictionary);
+}
 
+function InventoryItemFeetHempRopeNpcDialog(C, Option) {
```
Comment (ace-1331) [inline BondageClub/Screens/Inventory/ItemPelvis/HempRope/HempRope.js:38]: The name of the function here does not refer to the correct item.

File: BondageClub/Screens/Inventory/ItemLegs/HempRope/HempRope.js [hunk 2]
```diff
@@ -3,122 +3,51 @@
...
+	var msg = "LegRopeSet" + Option.Name;
+	var Dictionary = [];
+	Dictionary.push({ Tag: "SourceCharacter", Text: Player.Name, MemberNumber: Player.MemberNumber });
+	Dictionary.push({ Tag: "TargetCharacter", Text: C.Name, MemberNumber: C.MemberNumber });
+	ChatRoomPublishCustomAction(msg, true, Dictionary);
+}
 
-}
\ No newline at end of file
+function InventoryItemFeetHempRopeNpcDialog(C, Option) {
```
Comment (ace-1331) [inline BondageClub/Screens/Inventory/ItemLegs/HempRope/HempRope.js:51]: Same here, the name of the function is not the right one

**Review by ace-1331 — CHANGES_REQUESTED** [review]

**Review by ace-1331 — APPROVED** [review]

**Comment by ace-1331** [comment]
Sorry for the commits, I was testing a few things among what we can do right now. Looks like we can fix small mistakes like these then push to the branch without merging when appropriate to save some time.

This should help to prevent delays when handling PRs due to merge conflicts and such.
All looks good to me now!

**Review by ace-1331 — APPROVED** [review]

**Review by elliesec — APPROVED** [review]
Yep, all looks good to me!

**Comment by elliesec** [comment]
Well this got a whole lot smaller 😅

**Pull request merged**